TG-Netflix/src/components/player/BigModalPlayer.jsx

```diff
@@ -0,0 +1,165 @@
+import React, { useEffect, useState, useRef } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import YouTube from 'react-youtube';
+
+// CSS & BUTTONS
+import { PlayerStyles } from './BigModalPlayer.style';
+
+import play from '../../assets/play-button.svg';
+import volume from '../../assets/volume.svg';
+import muted from '../../assets/muted.svg';
+// import { setTrailerTime } from '../../reducers/trailerReducer';
+import { useSearchParams } from 'react-router-dom';
+
+function BigModalPlayer() {
+    const { overview, backdrop_path, trailer, title, id } = useSelector(state => state.netflix.browse[1])
+    console.log(id)
+
+    // const seconds = useSelector((state) => state.trailer.seconds)
+
+    const dispatch = useDispatch();
+    // REF
+    const modalPlayerRef = useRef(null);
+    const bannerRef = useRef(null);
+    const bannerTitleRef = useRef(null);
+    const bannerDescRef = useRef(null);
+    const playButtonRef = useRef(null);
+
+    // VOLUME STATE
+    const [muteIsVisible, setMuteIsVisible] = useState(true);
+    const [unMuteIsVisible, setUnMuteIsVisible] = useState(false);
+
+    // ANIMATION
+    const [isPlaying, setIsplaying] = useState(false)
+
+    // MODAL STATE
+    const [movieID, setMovieID] = useSearchParams();
```

Comment:
Line 20, 33 and 36 seems to not be used while viewing in VSCode. Is dulled out, check it's usage.